content/2017/1/3/initial-letter.rst

```diff
@@ -28,7 +28,7 @@ append the ``::first-letter`` pseudo element. The property is
 ``initial-letter`` and the value is the number of lines you would like your
 initial letter to expand.
 
-.. code-block:: css
+.. code-block:: scss
```

Comment:
curious, is there a reason we changed this to sass instead of css? Almost all blocks here are CSS except the 1 that uses a variable. Wasn't sure if it is required for the highlighting or something else that I should be aware of.